lxd/instance/instance_utils.go

```diff
@@ -125,8 +125,43 @@ func exclusiveConfigKeys(key1 string, key2 string, config map[string]string) (va
 	return "", false, nil
 }
 
+var containerSpecificConfigKeys = []string{
+	"security.nesting",
+	"limits.cpu.allowance",
+	"limits.cpu.priority",
+	"limits.hugepages",
+	"limits.kernel",
+	"limits.memory.enforce",
+	"limits.memory.swap",
+	"limits.processes",
+	"linux.kernel_modules",
+	"migration.incremental.memory",
+	"migration.incremental.memory.goal",
+	"migration.incremental.memory.iterations",
+	"nvidia.driver.capabilities",
+	"nvidia.runtime",
+	"nvidia.require.cuda",
+	"nvidia.require.driver",
+	"raw.idmap",
+	"raw.lxc",
+	"raw.seccomp",
+	"security.devlxd",
+	"security.idmap",
+	"security.nesting",
+	"security.privileged",
+	"security.protection.shift",
+	"security.syscalls",
+	"security.privileged",
+	"limits.memory.swap.priority"}
```

Comment:
Trailing `}` needs to be on newline (did you run this through `gofmt` btw?).

Reply:
I usually run gofmt -s -w ./.. before I commit something. I ran it again for this file, but it seems that it does not realign this bracket.

Reply:
@my5t3ry ah ok in that case its just our own project specific convention then :)